src/gui/Assignments.js

```diff
@@ -0,0 +1,36 @@
+
+/* global document */
+
+class Assignment {
+
+    constructor(name, link, dueDate) {
+        this.aName = name;
+        this.aLink = link;
+        this.dueDate = dueDate;
+    }
+}
+
+const assignments = [
+    new Assignment("Deliverable 2", "", new Date(2019, 10, 4)),
+    new Assignment("Deliverable 3", "", new Date(2019, 11, 3)),
+    new Assignment("Final Test", "", new Date(2019, 10, 15)),
+];
+
+/*
+ Takes an array of assignment objects
+ */
+async function populateGrades() {
+    const alist = document.getElementById("assignment_grades");
+    assignments.sort((a, b) => a.dueDate.getTime() - b.dueDate.getTime());
+    for (let i = 0; i < assignments.length; ++i) {
+        const asgn = assignments[i];
+        const date = asgn.dueDate;
+        alist.insertAdjacentHTML("beforeend", `
```

Comment:
inserting html is typically frowned upon because of HTML injection. Probably not an issue for what we are doing, but I am writing this in case we decide it is